lib/avromatic/model/serialization.rb

```diff
@@ -50,38 +42,69 @@ def avro_hash(fields)
                                end
           end
         end
+
+        def encode(data, key_or_value = :value)
```

Comment:
This module seems to have quite a few methods that will be added to generated models. Should they either be prefixed with `avro_` or put in a separate helper module that is not included in generated models?